core/src/main/java/io/aiven/klaw/service/SchemaRegstryControllerService.java

```diff
@@ -270,6 +276,82 @@ public ApiResponse execSchemaRequestsDecline(String avroSchemaId, String reasonF
     }
   }
 
+  public ApiResponse promoteSchema(SchemaPromotion schemaPromotion) throws Exception {
+    String userDetails = getUserName();
+
+    if (commonUtilsService.isNotAuthorizedUser(
```

Comment:
can we add one more validation if the user belongs to the same team as topic owner